### PR TITLE
Add MCC/MNC support

### PR DESCRIPTION
**Pull request by oschwald**
- Update test databases
- Add MCC/MNC support
- Replace unicode type with str in docs
- Test on 3.10 and list it as supported

### CODE REVIEW

File: geoip2/models.py [hunk 1]
```diff
@@ -577,6 +577,23 @@ class ISP(ASN):
...
+
+      The `mobile country code (MCC)
+      <https://en.wikipedia.org/wiki/Mobile_country_code>`_associated with the
+      IP address and ISP.
+
+      :type: str
+
+    .. attribute: mobile_network_code
+
+      The `mobile network code (MCC)
```
Comment (horgh) [inline geoip2/models.py:590]: "MNC"?

File: geoip2/models.py [hunk 2]
```diff
@@ -577,6 +577,23 @@ class ISP(ASN):
...
+      <https://en.wikipedia.org/wiki/Mobile_country_code>`_associated with the
+      IP address and ISP.
+
+      :type: str
+
+
```
Comment (horgh) [inline geoip2/models.py:596]: Is there an extra newline? I guess it doesn't matter though :-P.

File: geoip2/models.py [hunk 3]
```diff
@@ -577,6 +577,23 @@ class ISP(ASN):
...
+      The `mobile country code (MCC)
+      <https://en.wikipedia.org/wiki/Mobile_country_code>`_ associated with the
+      IP address and ISP.
+
+      :type: str
+
+    .. attribute: mobile_network_code
+
+      The `mobile network code (MCC)
+      <https://en.wikipedia.org/wiki/Mobile_country_code>`_ associated with the
```
Comment (horgh) [inline geoip2/models.py:591]: Is the URL correct?

File: geoip2/records.py [hunk 4]
```diff
@@ -726,6 +726,24 @@ class Traits(Record):
...
+      <https://en.wikipedia.org/wiki/Mobile_country_code>`_ associated with the
+      IP address and ISP. This attribute is available from the City and
+      Insights web services and the GeoIP2 Enterprise database.
+
+      :type: str
+
+    .. attribute: mobile_network_code
+
+      The `mobile network code (MCC)
+      <https://en.wikipedia.org/wiki/Mobile_country_code>`_ associated with the
```
Comment (horgh) [inline geoip2/records.py:741]: Same comments as above

**Review by horgh — CHANGES_REQUESTED** [review]

File: geoip2/models.py (hunk 3 above)
Comment (oschwald) [reply geoip2/models.py:591]: Yes, I think it is. Miguel provided it and I didn't see a better one.

**Review by horgh — APPROVED** [review]

**Pull request merged**